src/DynamoCore/Graph/Nodes/NodeModel.cs

```diff
@@ -1129,13 +1110,13 @@ public MirrorData GetValue(int outPortIndex, EngineController engine)
         }
 
         /// <summary>
-        ///     Sets the nickname of this node from the attributes on the class definining it.
+        ///     Sets the name of this node from the attributes on the class definining it.
         /// </summary>
-        public void SetNickNameFromAttribute()
+        public void SetNamePropertyFromAttribute()
```

Comment:
Can we align the two functions' name `SetNamePropertyFromAttribute` and `getNameFromNodeNameAttribute`?